tests/test_heatpump.py

```diff
@@ -1,17 +1,44 @@
 import pytest
 
-from modules.class_heatpump import Heatpump
+from modules.heatpump import HeatPump
 
 
-@pytest.fixture(scope="function")
-def heatpump() -> Heatpump:
-    """Heatpump with 5 kw heating power and 24 h prediction"""
-    return Heatpump(5000, 24)
+class TestHeatPump:
+    def test_cop(self, hp_5kw_24h: HeatPump):
+        """Testing calculate COP for variouse outside temperatures"""
+        assert hp_5kw_24h.calculate_cop(-10) == 2.0, "COP for -10 degree isn't correct"
+        assert hp_5kw_24h.calculate_cop(0) == 3.0, "COP for 0 degree isn't correct"
+        assert hp_5kw_24h.calculate_cop(10) == 4.0, "COP for 10 degree isn't correct"
```

Comment:
the messages are trivial, given that the assert statement is already quite self-explanatory
```suggestion
        assert hp_5kw_24h.calculate_cop(-10) == 2.0
        assert hp_5kw_24h.calculate_cop(0) == 3.0
        assert hp_5kw_24h.calculate_cop(10) == 4.0
```